packages/ble-transport/src/transports/BleTransportSession.ts

```diff
@@ -0,0 +1,31 @@
+import type { Central } from '@animo-id/react-native-ble-didcomm'
+import type { EncryptedMessage, Logger, TransportSession } from '@aries-framework/core'
+
+import { ConsoleLogger, LogLevel, JsonEncoder } from '@aries-framework/core'
+
+export class BleTransportSession implements TransportSession {
+  public readonly type = 'Bluetooth Low Energy'
+  public id: string
+  private logger: Logger
+  private central: Central
+
+  public constructor(id: string, sdk: Central) {
+    this.id = id
+    this.logger = new ConsoleLogger(LogLevel.debug)
```

Comment:
Should not use a custom logger here. We can add it in the constructor if needed.